source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb

```diff
@@ -1,187 +1,95 @@
+If __Test Mode__ is at the bottom of the left-hand menu, switch to __Production Mode__. Worldpay uses the term 'production' for live accounts.
 
-## Set up your profile
+## Connect your Worldpay account to GOV.UK Pay
 
-To set up your profile, you need to copy credentials from your Worldpay account
-into your GOV.UK Pay account.
+1. Select your live account from the [__My services__ page](https://selfservice.payments.service.gov.uk/my-services) in the GOV.UK Pay admin tool.
 
-You must separately sign in to both your [Worldpay
-account](https://secure.worldpay.com/sso/public/auth/login.html) and your
-[GOV.UK Pay account](https://selfservice.payments.service.gov.uk/login). You
-should have both accounts open, in different web browser tabs.
+2. Select [__Your PSP - Worldpay__].
 
-1. In your Worldpay browser tab, select __Profile__ in the left-hand menu. If you
-are prompted to choose a merchant code, select the appropriate one.
+3. In your Worldpay account, select __Account__, then the __Profile__ tab. Select your merchant code if you're asked for one.
 
-1. In your GOV.UK Pay browser tab, go to __My services__ to select the live
-service you want to set up. Select __Settings__, then __Your PSP - Worldpay__,
-then __Change__ under __Account credentials__.
+4. Copy the __Merchant Code__ value from the __Identification__ section of your Worldpay account into the __Merchant code__ field in your GOV.UK Pay account.
 
-1. In Worldpay, copy the Worldpay __Merchant Code__ from __Identification__ into
-the GOV.UK Pay __Merchant code__ field.
+5. Copy the __New Username__ value from your Worldpay account into the __Username__ field in your GOV.UK Pay account.
 
-1. Copy the Worldpay __New Username__ into the GOV.UK Pay __Username__ field.
-You should see the option __Enable Original XML Username__. If you see the
-option __Disable Original XML Username__, you must select the checkbox and then
-the __Save Profile__ button at the bottom of the page)
+6. If you see a checkbox labelled __Disable Original XML Username__, select the checkbox then
+select __Save Profile__ at the bottom.
 
-1. Change the Worldpay XML password:
+7. If you have not set your XML password for this merchant code before, set it by selecting the pencil icon next to __XML Password__ and choosing a password. Confirm by selecting __Add new password__, then __OK__. Save the password by selecting the pencil icon again, then __Complete__, then __OK__.
 
-    <br> select the pencil icon next to __XML Password__ </br>
-    <br> change the password and select __Add new Password__ </br>
-    <br> select __OK__ to go back to the __Profile__ page </br>
-    <br> select the pencil icon again, then __Complete__ then __OK__ </br>
-    <br>
+    > If you've set your XML password before but you cannot remember it, you can set a new password. But you must add the new password to your other services that use Worldpay or they'll stop working.
```

Comment:
Maybe be more specific and say “this Worldpay account” (or “merchant code” or whatever)?